code/modules/mob/living/simple_animal/hostile/gym_rats.dm

```diff
@@ -99,6 +101,15 @@
 		playsound(src, 'sound/items/eatfood.ogg', rand(10,50), 1)
 		if(maxHealth < health_cap) // Are we below our max gainz level? Add on some max hp!
 			adjust_hp(5)
+			if(rattype = regular) //regular gym rat
```

Comment:
rattype = regular -> rattype == regular

Reply:
ah that makes sense, forgot about double trues